docs/en/observability/apm/known-issues.asciidoc

```diff
@@ -43,11 +43,12 @@ lifecycle configurations.
 Upgrading to 8.15.1 should fix any new indices created for the data stream. However,
 indices created in version 8.15.0 would remain unmanaged if the default ILM policy is
 used. One of the following approaches can be adopted to fix the unmanaged indices:
-1. Manually delete the indices when they are no longer needed.
-2. Explicitly configure APM data streams with the default data stream lifecycle config.
+
+. Manually delete the indices when they are no longer needed.
+. Explicitly configure APM data streams with the default data stream lifecycle config.
```

Comment:
[**For reviewers**] Not sure what I am doing wrong here, the points are not enumerated even after this change: https://observability-docs_bk_4226.docs-preview.app.elstc.co/guide/en/observability/current/apm-known-issues.html

Reply:
This PR only updates the `main` branch, which maps to the `/master/` URL: https://observability-docs_bk_4226.docs-preview.app.elstc.co/guide/en/observability/master/apm-known-issues.html. It seems to be fixed there! It will be fixed on `/current/` the backport to 8.15 is published.

Reply:
Ah, thanks!